test/commentRouterTest.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+require('dotenv').load();
+const superagent = require('superagent');
+const storage = require('../lib/storage');
+const expect = require('chai').expect;
+const Comment = require('../model/comment.js');
+const baseUrl = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+describe('testing comment router', function(){
+  describe('testing POST /api/comments', function(){
+    after((done) => {
+      Comment.deleteById(this.tempComment.id)
+      .then(() => done())
+      .catch(done);
+    });
+
+    it('should create comment', (done) => {
+      superagent.post(`${baseUrl}/api/comments`)
+      .send({
+        announcer: 'Troy Aikman',
+        comment: 'He did a thing',
+        year: '2017',
+      })
+      .then(res => {
+        console.log(res.body);
+        console.log(res.statusCode);
+        expect(res.status).to.equal(200);
+        expect(res.body.announcer).to.equal('Troy Aikman');
+        expect(res.body.comment).to.equal('He did a thing');
+        expect(res.body.year).to.equal('2017');
+        expect(Boolean(res.body.id)).to.equal(true);
+        this.tempComment = res.body;
+        done();
+      })
+      .catch(done);
+    });
+
+    it('should return 400 status code', (done) => {
+      superagent.post(`${baseUrl}/api/comments`)
+      .send({
+        announcer: 'Troy Aikman',
+      })
+      .then(done)
+      .catch(err=> {
+        console.log(err.status);
+        expect(err.status).to.equal(400);
+        done();
+      })
+      .catch(done);
+    });
+  });
+});
+
+describe('testing DELETE aka thats so fetch', function(){
```

Comment:
Your entire describe block for ``DELETE`` is a false positive. You have no ``expect`` statements going on, so it just passes automatically by default. 

You're not losing points since a ``DELETE`` test wasn't a requirement for lab. Nevertheless, it's cool that you still wrote one because that IS an important route to test. You just need to write it so you make sure it works. :)